### PR TITLE
Create an access log when user views document in the hub

### PR DESCRIPTION
**Pull request by jenny-heath**
This is for a small fry ticket to create an access log when users view a document. We created this after doing a threat modeling exercise and realizing that documents have the most potential to contain the most sensitive information clients give to us.

Part of this story included making the client association on `access_log` polymorphic to include documents as an associated record.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -111,6 +112,19 @@
     t.index ["name"], name: "index_coalitions_on_name", unique: true
   end
 
+  create_table "consents", force: :cascade do |t|
```
Comment (bytheway875) [inline db/schema.rb:115]: I think you've got a dirty DB from the other ticket you were working on! Might need to drop your db, or can just manaully remove this from the schema.

**Review by bytheway875 — CHANGES_REQUESTED** [review]
Dirty schema!

**Comment by bytheway875** [comment]
Two notes: 

- This won't capture views from the edit page using the Greeter bookmarklet, so we won't really be getting full coverage access logs. 😢 
- If/when we eventually add viewing the document from the edit page, we'll need to add an access log there as well I think? Might be worth updating that ticket to reflect that.

File: db/schema.rb (hunk 1 above)
Comment (jenny-heath) [reply db/schema.rb]: good catch!!

**Comment by bytheway875** [comment]
👍

**Pull request merged**